common/taskana-common/src/test/java/pro/taskana/common/api/WorkingTimeCalculatorTest.java

```diff
@@ -212,4 +60,98 @@ void should_ReturnMultipleWorkingTimes_When_CalculatorUsedMultipleTimes() throws
 
     assertThat(duration2).isEqualTo(Duration.of(56, ChronoUnit.HOURS));
   }
+
+  @TestFactory
+  Stream<DynamicTest> should_ReturnWorkingTime() {
+    List<Quadruple<String, Instant, Instant, Duration>> valuesForTests =
+        List.of(
+            // Test instants that are within same day
+            Quadruple.of(
+                "Same day, within working hours",
```

Comment:
I know that the test description did exist previously.. But I don't like them at all. For me most of them seem confusing.
Suggestion: A short pairing session where we come up with better names :)